pkg/nfd-master/nfd-master.go

```diff
@@ -449,6 +451,17 @@ func verifyNodeName(cert *x509.Certificate, nodeName string) error {
 	return nil
 }
 
+func isNamespaceDenied(labelNs string, deniedNs map[string]struct{}) bool {
+	for ns := range deniedNs {
+		deniedNsWildcard := fmt.Sprintf("*.%v", ns)
```

Comment:
I don't think we want to forcibly make all items a glob. Otherwise you couldn't do something like "deny `kubernetes.io` but allow all `*.kubernetes.io`"

Reply:
Yes true, but I just followed what was said [here.](https://github.com/kubernetes-sigs/node-feature-discovery/issues/1022#issuecomment-1412215582) Which means basically denying all namespaces. In fact, the user will input a namespace (without wildcard) and we will deny all sub-namespaces. 
However, If you see that we need to give users the flexibility to deny all sub-namespaces using wildcards, I think we should change this implementation above.

Reply:
It can be that what I suggested above is not the "right thing to do". NOT forcing a glob (i.e. not automatically denying sub-namespaces) would give us the most flexibility. I now feel that if we forcibly deny all sub-namespaces it will bit us later (could be countered by supporting wildcards in `-extra-label-ns` though)

Let's see what others think. Any thoughts on this @cedricmckinnie @ArangoGutierrez @zvonkok @fmuyassarov ?

Reply:
Yes, I agree with you!